pkg/apic/apiserver/models/api/v1/resourcemeta.go

```diff
@@ -18,7 +18,7 @@ type Meta interface {
 // ResourceMeta metadata for a ResourceInstance
 type ResourceMeta struct {
 	GroupVersionKind
-	Name  string `json:"name"`
+	Name  string `json:"name,omitempty"`
```

Comment:
Can you add/update a test in resourcemeta_test.go for a resource without a name?

Reply:
will do